### ISSUE TITLE
Error with Installing GraphViz/PsGraph

### ISSUE THREAD

**Issue by oliwex** [KevinMarquette/PSGraph#107]
Hi i try to install PsGraph and graphviz and have problem. Try this two sollutions:

#1.First way install GraphViz

# Install GraphViz from the Chocolatey repo
Register-PackageSource -Name Chocolatey -ProviderName Chocolatey -Location http://chocolatey.org/api/v2/
Find-Package graphviz | Install-Package -ForceBootstrap

# Install PSGraph from the Powershell Gallery
Find-Module PSGraph | Install-Module

# Import Module
Import-Module PSGraph

######################################################################################################################
#2.Second way install GraphViz
#idea: https://github.com/PrateekKumarSingh/AzViz/issues/12
#Set TLS 1.3
New-Item 'HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Server' -Force | Out-Null
New-ItemProperty -path 'HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Server' -name 'Enabled' -value '0' -PropertyType 'DWord' -Force | Out-Null
New-ItemProperty -path 'HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Server' -name 'DisabledByDefault' -value 1 -PropertyType 'DWord' -Force | Out-Null
New-Item 'HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client' -Force | Out-Null
New-ItemProperty -path 'HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client' -name 'Enabled' -value '0' -PropertyType 'DWord' -Force | Out-Null
New-ItemProperty -path 'HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client' -name 'DisabledByDefault' -value 1 -PropertyType 'DWord' -Force | Out-Null

#Change .NET default TLS Version
reg add HKLM\SOFTWARE\Microsoft\.NETFramework\v4.0.30319 /v SystemDefaultTlsVersions /t REG_DWORD /d 1 /f /reg:64
reg add HKLM\SOFTWARE\Microsoft\.NETFramework\v4.0.30319 /v SystemDefaultTlsVersions /t REG_DWORD /d 1 /f /reg:32

#Install chocolatey
iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))

#Install GraphViz
choco install graphviz

################################################################################################

To sum up none of this works:

After using the second one i must manually install: C:\Chocolatey\lib\Graphviz.2.47.3\tools\stable_windows_10_cmake_Release_x64_graphviz-install-2.47.3-win64.exe.exe

And then it works.

Could you please update Install-module cmdlet to properly install graphviz and psgraph?

**Comment by KevinMarquette** [comment]
Thank you, I will review the process. 

One issue with this project is that it depends on graphviz as an external application and `Install-Module` does not support the functionality to install it. So the graphviz install will always be its own process. 

I was recommending the use of Chocolatey for installing that package, but if it has become that complicated, then I will re-evaluate that guidance.

**Comment by oliwex** [comment]
So maybe install-module function can install chocolatey and the rest, because it can be better for newbie devs
